README.md

```diff
@@ -24,50 +24,53 @@ This software is a personal project and not related with any companies, includin
 ## Usage
 ```
 $ ./ysoserial -h
-Missing arguments.
 ysoserial.net generates deserialization payloads for a variety of .NET formatters.
 
 Available gadgets:
 
 	ActivitySurrogateDisableTypeCheck (Disables 4.8+ type protections for ActivitySurrogateSelector, command is ignored.)
 		Formatters:
-			BinaryFormatter, ObjectStateFormatter, SoapFormatter, NetDataContractSerializer, LosFormatter
+			BinaryFormatter, LosFormatter, NetDataContractSerializer, ObjectStateFormatter, SoapFormatter
 
-	ActivitySurrogateSelectorFromFile (Another variant of the ActivitySurrogateSelector gadget. This gadget interprets the command parameter as path to the .cs file that should be compiled as exploit class. Use semicolon to separate the file from additionally required assemblies, e. g., '-c ExploitClass.cs;System.Windows.Forms.dll'.)
+	ActivitySurrogateSelector (This gadget ignores the command parameter and executes the constructor of ExploitClass class.)
 		Formatters:
-			BinaryFormatter, ObjectStateFormatter, SoapFormatter, LosFormatter
+			BinaryFormatter, LosFormatter, ObjectStateFormatter, SoapFormatter
 
-	ActivitySurrogateSelector (This gadget ignores the command parameter and executes the constructor of ExploitClass class.)
+	ActivitySurrogateSelectorFromFile (Another variant of the ActivitySurrogateSelector gadget. This gadget interprets the command parameter as path to the .cs file that should be compiled as exploit class. Use semicolon to separate the file from additionally required assemblies, e. g., '-c ExploitClass.cs;System.Windows.Forms.dll'.)
 		Formatters:
-			BinaryFormatter, ObjectStateFormatter, SoapFormatter, LosFormatter
+			BinaryFormatter, LosFormatter, ObjectStateFormatter, SoapFormatter
 
 	ObjectDataProvider (ObjectDataProvider gadget)
 		Formatters:
-			Xaml, Json.Net, FastJson, JavaScriptSerializer, XmlSerializer, DataContractSerializer, YamlDotNet < 5.0.0, FsPickler
+			DataContractSerializer, FastJson, FsPickler, JavaScriptSerializer, Json.Net, Xaml, XmlSerializer, YamlDotNet < 5.0.0
 
-	TextFormattingRunProperties (TextFormattingRunProperties gadget)
+	PSObject (PSObject gadget. Target must run a system not patched for CVE-2017-8565 (Published: 07/11/2017))
 		Formatters:
-			BinaryFormatter, ObjectStateFormatter, SoapFormatter, NetDataContractSerializer, LosFormatter
+			BinaryFormatter, LosFormatter, NetDataContractSerializer, ObjectStateFormatter, SoapFormatter
 
-	PSObject (PSObject gadget. Target must run a system not patched for CVE-2017-8565 (Published: 07/11/2017))
+	SessionSecurityToken (SessionSecurityTokenGenerator (System.IdentityModel.Tokens namespace) gadget)
```

Comment:
do we need the namespace info here?

Reply:
I just didn't want to make it too confusing - I also think we may want to add this info as an additional bit to all the gadgets so people would know which namespace they are dealing with? We can always remove it in the future releases if we find it useless.